src/camvidlog2/moondream/ai.py

```
@@ -0,0 +1,82 @@
+import time
+from typing import Iterable
+
+import numpy as np
+import pandas as pd
+from PIL import Image
+from supervision import ByteTrack, Detections, DetectionsSmoother
+from transformers import AutoModelForCausalLM
+
+
+def generate_bboxes(
```

Ruff (0.12.2)

Missing return type annotation for public function `generate_bboxes`

(ANN201)